packages/grafana-ui/src/components/DataLinks/FieldLink.tsx

```diff
@@ -0,0 +1,64 @@
+import { Field, GrafanaTheme, LinkModel } from '@grafana/data';
+import { css } from 'emotion';
+import React from 'react';
+import { useStyles } from '../../themes';
+import { Icon } from '../Icon/Icon';
+import { DataLinkButton } from './DataLinkButton';
+
+type Props = {
+  links: Array<LinkModel<Field>>;
+};
+
+export function FieldLink({ links }: Props) {
+  const styles = useStyles(getStyles);
+
+  if (links.length === 1) {
+    return <DataLinkButton link={links[0]} />;
+  }
+
+  const externalLinks = links.filter((link) => link.target === '_blank');
+  const internalLinks = links.filter((link) => link.target === '_self');
+
+  return (
+    <>
+      {internalLinks.map((link, i) => {
+        return <DataLinkButton key={i} link={link} />;
```

Comment:
So previously `DataLinkButton` was meant to work for both internal and external links. Does this separation now mean we it should not be used for external links? In that case maybe we should have a `InternalDataLinkButton` and `ExternalDataLinkButton` to make that clear?

Reply:
Not sure. I think if we only have external link then it makes sense to use the primary button which is now the DataLinkButton but other than use a simple link? WDYT @jessover9000 ?

Reply:
We can also use the primary button for external links, but with the current external link text (like "Go to [long url with lots of characters]"), the button might become very long and therefore the overlay window might become quite chunky too. How do we usually handle long link texts in a button? Do we have a solution for that built into the button component (line breaks or shortening the text)?